src/main/java/frc/robot/Limelight.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot;
+
+import org.photonvision.PhotonCamera;
+import edu.wpi.first.math.util.Units;
+import frc.robot.Constants.VisionConstants;
+
+public class Limelight {
+	private PhotonCamera photonCamera;
+	private int currPipelineIndex;
+	/** Intializes Limelight object. */
+	public Limelight() {
+		photonCamera = new PhotonCamera("OV5647");
```

Comment:
You're opening the PhotonCamera twice, here and in the AprilTag constructor. It might be better to create the PhotonCamera on its own and pass it in as an argument to these classes so we don't get the settings mixed up?